Updates Title Screen Instructions

ORCACommander opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

The instructions Found in the readme file for config directory loaded title screens has become out of date. The move to wide screen title cards invalidated the specification outlined in the readme. Also a section of the readme mentioned loading map names from .DM files. Those map names names were migrated to JSON files long ago and the documentation in the readme was never updated to reflect it.

## Why It's Good For The Game

Correct documentation on title card specifications leads to more efficient work flows as les time is pent on correction.

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
qol: Updated Title Card Specifications
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

ORCACommander committed:
Updates Title Screen Instructions

Fikou commented on config/title_screens/README.txt:
```diff
@@ -1,7 +1,10 @@
+
+Specification:
+Formats: PNG, JPG, GIF, DMI*
+Dimensions: 480x544
```
what?

ORCACommander replied:
I didn't sleep well aslt night...

ORCACommander replied:
as for 480^2

Sure if you like letter boxing....

ORCACommander committed:
I need more sleep

Fikou commented on config/title_screens/README.txt:
```diff
@@ -1,7 +1,10 @@
+
+Specification:
+Formats: PNG, JPG, GIF, DMI*
+Dimensions: 544x480
```
its still incorrect widescreen is not 544

ORCACommander replied:
and, e-mail reply did not atatch it to the thread >_<

The title screen is overlayed on a special area of the centcom map. the dimensions of this special area determine the pixel dimensions of the title card. Current standard for TG for a tile is 32x32.  this section of the map is 17 tiles wide by 15 tiles tall. thus 544x480

ORCACommander replied:
well that explains why that turf is out of bounds in strong dmm  *facepalms*

ORCACommander commented:
32 pixel width per tile, 17 tiles, 544 pixels

On Tue, Mar 1, 2022, 4:38 PM Fikou ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In config/title_screens/README.txt
> <https://github.com/tgstation/tgstation/pull/65233#discussion_r817160703>:
>
> > @@ -1,7 +1,10 @@
> -The enclosed /images folder holds the image files used as the title screen for the game. All common formats such as PNG, JPG, and GIF are supported.
> -Byond's DMI format is also supported, but if you use a DMI only include one image per file and do not give it an icon_state (the text label below the image).
> +The enclosed /images folder holds the image files used as the title screen for the game.
> +
> +Specification:
> +Formats: PNG, JPG, GIF, DMI*
> +Dimensions: 544x480
>
> its still incorrect widescreen is not 544
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tgstation/tgstation/pull/65233#pullrequestreview-896850073>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAZ2MV3IDU3MTOBVGW4IXX3U52E5HANCNFSM5PU74JCQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

ORCACommander committed:
APARENTLY i AM WONG.

There are 2 more columns of tiles reportedly that exist out of bounds that Strong Dmm can't display properly

ORCACommander commented:
any further thoughts on this?
